### PR TITLE
Development

### PR DESCRIPTION
**Pull request by instylevii**
Here is my demo for WebDriver. There is one link that will still need to be updated once the details page is up.

### CODE REVIEW

File: webdriver/index.html [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+	<link rel="stylesheet" type="text/css" href="styles/demotemplate.css"/>
+	<link rel="stylesheet" type="text/css" href="styles/demo.css"/>
+</head>
+<body>
+	<article class="demo-wrap">
+		<div class="row">
+			<div class="col-sm-15">
+				<header class="demo-header">
+					<div class="container">
+						<heading class="section-header">
```
Comment (jacobrossi) [inline webdriver/index.html]: heading isn't a real element. I think you mean &lt;header&gt;

**Comment by jacobrossi** [comment]
All the dot files (e.g. .csslintrc) shouldn't be included in your commit. Can you remove them?

File: webdriver/scripts/demo.js [hunk 2]
```diff
@@ -0,0 +1,398 @@
+(function(){
+    "use strict";
+    
+    var commands;
+    var sessionId;
+    var elementId;
+    
+    window.onload = function setup() {
```
Comment (jacobrossi) [inline webdriver/scripts/demo.js]: Have you run `grunt lint`? I don't think "setup()" will pass. I think it needs to be "setup ()"

File: webdriver/scripts/demo.js [hunk 3]
```diff
@@ -0,0 +1,398 @@
...
+            o.id = "commands-select-" + commands[j].commandName;
+            o.innerHTML = commands[j].commandTitle;
+    
+            s.appendChild(o);
+        }
+    }
+    
+    function loadCommands() {
+        commands = [];
+        commands.push({
```
Comment (jacobrossi) [inline webdriver/scripts/demo.js]: This is an awkward way to build the array. We should probably use a literal here, e.g.

```
    commands = [ {commandName: "click",
            commandTitle: "Click",
            method: "POST",
            path: "/session/SESSION_ID/element/ELEMENT_ID/click",
            requestBody: "",
            description: "Clicks the specified element." 
    }, {
              commandName: "executeScript",
              commandTitle: "Execute JavaScript",
              method: "POST",
              path: "/session/SESSION_ID/execute",
              requestBody: "{\"script\": \"alert(arguments[0].first);\",\"args\": [{\"first\":\"hey\"}]}",
              description: "Runs script on the page." 
    },
```

etc...

**Comment by melanierichards** [comment]
@InstyleVII Sorry again about the confusion, will work with @molant to get that demo generator updated when he's back in. :]

Here's what we discussed:
- @supports and web audio tuner are good demos to yank from. In fact it might be a better idea to copy the HTML there and insert your content where applicable, just so nothing is missed.
- Most of the differences are in larger structural divs/sections, a couple type styles, button classes.
- You'll be using the external CSS reference from the current demos instead of the `demotemplate.css` you have included here.
- CSS/markup reference: http://edgecore.azurewebsites.net/documentation/pattern-library.html
- We can work together tomorrow to run the linter on your code. We actually need to merge your PR to do so.

**Pull request closed without merging**